bin/check_fixmystreet

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env perl
+
+use strict;
+use lib '/usr/lib/nagios/plugins';
+use utils qw(%ERRORS);
+use Getopt::Long;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../fixmystreet/setenv.pl";
+}
+
+my $reports = 50;
+my $updates = 0;
+
+GetOptions(
+    "reports=i" => \$reports,
+    "updates=i" => \$updates,
+);
+
+sub unsent_report_count {
+
+    use FixMyStreet::DB;
```

Comment:
use is run at compile time, so these and ones below might as well be just above where you set $reports and $updates.